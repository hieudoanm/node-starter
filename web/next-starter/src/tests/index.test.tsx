import { render } from '@testing-library/react';
import HomePage from '../pages';

describe('HomePage', () => {
  it('to match snapshot', () => {
    const { asFragment } = render(<HomePage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
