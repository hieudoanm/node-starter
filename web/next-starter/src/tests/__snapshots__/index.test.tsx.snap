// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage to match snapshot 1`] = `<DocumentFragment />`;
